examples/v1beta1/kind-cluster/deploy.sh

```diff
@@ -36,8 +36,8 @@ if [ -z "$(command -v kubectl)" ]; then
   exit 1
 fi
 
-# Step 1. Create Kind cluster with Kubernetes v1.22.9
-kind create cluster --image kindest/node:v1.22.9
+# Step 1. Create Kind cluster with Kubernetes v1.23.6
```

Comment:
Any specific reason for changing default?

Reply:
I upgraded the KinD Kubernetes version since we have upgraded Kubernetes dependencies to v0.23.
Also, according to [this doc](https://kubernetes.io/releases/#release-v1-22), K8s v1.22 reach EoL before we will release after the next Katib major version.